packages/web/config/ibc-assets.ts

```diff
@@ -27,12 +28,22 @@ export const IBCAssetInfos: (IBCAsset & {
     sourceChannelId: "channel-208",
     destChannelId: "channel-3",
     coinMinimalDenom: "uusdc",
-    depositUrlOverride:
-      "https://satellite.money/?source=ethereum&destination=osmosis&asset_denom=uusdc",
-    withdrawUrlOverride:
-      "https://satellite.money/?source=osmosis&destination=ethereum&asset_denom=uusdc",
     sourceChainNameOverride: "Ethereum",
     isVerified: true,
+    originBridgeInfo: {
```

Comment:
Only for USDC for now?

Reply:
Initially. We will gradually roll out others as we test and fix any issues that may arise.